pkg/component/observability/plutono/dashboards/garden/virtual-garden-etcd-dashboard.json

```diff
@@ -1138,7 +1138,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(container_fs_writes_bytes_total{namespace=\"kube-system\", pod=~\"etcd-(${cluster:pipe}).*\"}[1m])) by(pod, container)",
+              "expr": "sum(rate(container_fs_writes_bytes_total{namespace=\"shoot-control-plane\", pod=~\"etcd-(${cluster:pipe}).*\"}[1m])) by(pod, container)",
```

Comment:
Found a similar `kube-system` in the same file.
https://github.com/gardener/gardener/blob/fcb57f8c2795f64518c22e7eada690bcd842d8b6/pkg/component/observability/plutono/dashboards/garden/virtual-garden-etcd-dashboard.json#L1041
Is this correct or does it have to be changed too?

Reply:
Oh sorry, I missed during my review that this change is in the virtual etcd dashboard. I think this is incorrect because the renaming we did from `kube-system` to `shoot-control-plane` only concerned shoot metrics. The idea was to differentiate shoot metrics coming from the `kube-system` namespace in the shoot and metrics coming from the shoot control-plane namespace in the seed. 

I think this change was meant to be done in `pkg/component/observability/plutono/dashboards/shoot/owners/etcd-dashboard.json` as well as the other query.

Thanks for catching this :)

Reply:
I'm not sure how that slipped by me. Thanks for the catch @oliver-goetz! 🫶